charts/argo-apps/ci/default-values.yaml

```diff
@@ -0,0 +1,4 @@
+argocdNotifications:
+  enabled: true
+  values:
+    TODO: add values in values.yaml
```

Comment:
What's the idea behind this `TODO` line?

Reply:
tbh, it's mostly cargo culted from elsewhere